MoreLinq/CountDown.cs

```diff
@@ -65,15 +65,11 @@ static partial class MoreEnumerable
 
             IEnumerable<TResult> IterateList(IListLike<T> list)
             {
-                var countdown = Math.Min(count, list.Count);
+                var count = list.Count;
```

Comment:
This variable hides `CountDown` parameter `count`.

Reply:
Yep, working late again. 😆 Fixed with da13c3f0c25786c033807f120cbbf9c1c01d5176.